.nvmrc

```
@@ -0,0 +1 @@
+v23.11.1
```

```suggestion
v23.6.1
```

I think 23.6 is the earliest that will work. If we add this here, might as well support as much as possible.

note this is also in the repo and used by ci
https://github.com/did-coop/wallet-attached-storage-server/blob/main/.node-version
https://github.com/did-coop/wallet-attached-storage-server/blob/main/.github/workflows/ci.yml#L28

---

Incorporated :)